content/develop/concepts/configuration/theming-fonts.md

```diff
@@ -0,0 +1,213 @@
+
+`.streamlit/config.toml`:
+
+```toml
+[server]
+enableStaticServing = true
+
+[[theme.fontFaces]]
+family="tuffy"
+url="app/static/Tuffy-Regular.ttf"
+style="normal"
+weight=400
+[[theme.fontFaces]]
+family="tuffy"
+url="app/static/Tuffy-Bold.ttf"
+style="normal"
+weight=700
+[[theme.fontFaces]]
+family="tuffy"
+url="app/static/Tuffy-Italic.ttf"
+style="italic"
+weight=400
+[[theme.fontFaces]]
+family="tuffy"
+url="app/static/Tuffy-BoldItalic.ttf"
+style="italic"
+weight=700
+
+[theme]
+font="tuffy"
+```
+
+Directory structure:
+
+```none
+project_directory/
+├── .streamlit/
+│   └── config.toml
+├── static/
+│   ├── Tuffy-Bold.ttf
+│   ├── Tuffy-BoldItalic.ttf
+│   ├── Tuffy-Italic.ttf
+│   └── Tuffy-Regular.ttf
+└── streamlit_app.py
+```
+
+### Example 3: Define an alternative font with fallbacks
+
+If you don't want to download and host your font files with your app, you can point to externally hosted font files. If your files aren't hosted with your app, it's recommended to declare fallback fonts.
+
+<Important>
+
+If you configure your app to include any third-party integrations, including externally hosted fonts, your app may transmit user data (for example, IP addresses) to external servers. As the app developer, you are solely responsible for notifying your users about these third-party integrations, providing access to relevant privacy policies, and ensuring compliance with all applicable data protection laws and regulations.
+
+</Important>
+
+In your configuration file, wherever you declare a default font, you can use a comma-separated list of fonts instead. You can always include one of Streamlit's default fonts as a final fallback.
+
+- To specify a fallback font in Example 1, in `.streamlit/config.toml`, change the `[theme]` table to the following text:
+
+  ```toml
+  [theme]
+  font="noto-sans, sans-serif"
+  codeFont="noto-mono, monospace"
+  ```
+
+  This is the same configuration as in Example 1 except that Source Sans and Source Mono are declared as fallback fonts. You can define more than one fallback. When you declare a default font, the font (or comma-separated list of fonts) is passed to the CSS [`font-family`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family) property.
+
+<Tip>
+
+If any of your font family names contain spaces and you are declaring a fallback sequence, use inner quotes around the names. For example, if you name the font `"Noto Sans"`, use `font="'Noto Sans', sans-serif"` instead.
+
+</Tip>
+
+## Font size
+
+You can set the base font size for your app in pixels. You must specify the base font size as an integer. The following configuration is equivalent to the default base font size of 16 pixels:
+
+```toml
+[theme]
+baseFontSize=16
+```
+
+## Font colors
+
+Font color options are described in [Customize colors and borders in your Streamlit app](/develop/concepts/configuration/theming-customize-colors-and-borders#textcolor-and-linkcolor).
+
```

Comment:
I get why this is here but could also leave out since it's already mentioned on the other page. Also OK to leave it as is though.